### PR TITLE
Consolidate PS questions from FAQ

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes b/265338182

Changes proposed in this pull request: 
This will remove redundant FAQs from PS FAQ page and add to respective docs if not already present
- Move FLEDGE questions/answers to FLEDGE docs
- Move Privacy Sandbox questions to PS overview
- https://pr-5381-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/faq/
- https://pr-5381-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/fledge-api/
- https://pr-5381-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/fledge/
- https://pr-5381-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/overview/

### CODE REVIEW

File: site/en/docs/privacy-sandbox/overview/index.md [hunk 1]
```diff
@@ -179,3 +205,25 @@ core principles underlying the APIs.
...
+
+The article series links to the matching monthly updates to the [Privacy Sandbox timeline](https://privacysandbox.com/timeline/) which shows the current status and schedule for proposals.
+
+These high-level resources will provide signposts to changes across the project, but for individual proposals where you want to follow in detail you should:
+
+ - Watch or Star proposal repos on GitHub to get notification of new issues and updates: the Privacy Sandbox [status page](/docs/privacy-sandbox/status/) provides a link to the repo for each proposal
+ - Join the associated [W3C group](https://www.w3.org/groups/) for regular meetings discussing the proposal detail
+ - Star the associated entry on [Chrome Platform Status](https://chromestatus.com) for email updates on Chrome implementation changes.
+
+ ### Get involved
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/overview/index.md:221]: Spacing here may break the markdown!

Comment (nancymic2) [reply site/en/docs/privacy-sandbox/overview/index.md:221]: I'm not seeing it in staging but maybe I'm looking in the wrong place. Where would it be broken? thanks!

File: site/en/docs/privacy-sandbox/fledge-api/index.md [hunk 2]
```diff
@@ -34,6 +34,16 @@ the browser, to choose relevant ads for websites the user has previously visited
...
+### Can the Topics API be used with the FLEDGE API? 
+
+Yes. An observed topic for the current user, provided by the Topics API, could be used as 
+contextual information by a seller or bidder. A topic could be included in
+the following properties:
+
+*  `auctionSignals`, a property of the auction configuration object passed to `navigator.runAdAuction()`
+*  `userBiddingSignals`, a property of the interest group configuration
+   object passed to `navigator.joinAdInterestGroup()`
+
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/fledge-api/index.md:46]: This is too high up in this document, which gives the information a little too much priority. Can you move this to a accordion section between "[What is the difference between FLEDGE and TURTLEDOVE?](https://pr-5381-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/fledge-api/#what-is-the-difference-between-fledge-and-turtledove)" and "[Available browser configuration](https://pr-5381-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/fledge-api/#available-browser-configuration)"?

Also, add a link to the Topics docs to the first mention of Topics API.

File: site/en/docs/privacy-sandbox/fledge/index.md [hunk 3]
```diff
@@ -475,3 +480,8 @@ We've written an  [API developer guide](/docs/privacy-sandbox/fledge-api) and bu
    Group](https://www.w3.org/community/web-adv/participants).
 -  **Developer support**: Ask questions and join discussions on the
    [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).
+-  **Current implementation**: For questions about the implementation currently available to test in
+   Chrome: [file a Chromium bug](https://bugs.chromium.org/p/chromium/issues/list?q=fledge).
+-  **Best practices**: For implementation, integration, and general best practice questions:
+   [create an issue](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support) 
+   on the Privacy Sandbox developer support repo.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/fledge/index.md:487]:
```suggestion
-  **Developer support**: Ask questions about implementation, and best practices, or join discussions on the
   [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).
-  **Current implementation**: For questions about the implementation currently available to test in
   Chrome: [file a Chromium bug](https://bugs.chromium.org/p/chromium/issues/list?q=fledge).
```

Comment (heyawhite) [reply site/en/docs/privacy-sandbox/fledge/index.md:487]: I'd rather not link out to the same place twice as though it's two different places, so this groups those bullet points together.

File: site/en/docs/privacy-sandbox/overview/index.md [hunk 4]
```diff
@@ -119,12 +119,38 @@ without passive tracking.
...
+stakeholders, have offered more than 30 proposals to date. These proposals can
+be found in the
+[public resources of W3C groups](https://github.com/w3c/web-advertising#ideas-and-proposals-links-outside-this-repo)
+and cover a wide variety of use cases and requirements.
+
+400+ participants have joined W3C groups to provide input including the
 [Improving Web Advertising Business Group](https://www.w3.org/community/web-adv/participants) and
 the [Privacy Community Group](https://www.w3.org/community/privacycg/participants).
-* Five API implementations available for testing in Chrome.
+Five API implementations are currently available for testing in Chrome.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/overview/index.md:131]:
```suggestion
## Where are the Privacy Sandbox APIs available?

Five API implementations are currently available for testing in Chrome.
```

Although this was in the "Who is working on Chrome" section, it really deserves its own heading about implementation.

File: site/en/docs/privacy-sandbox/overview/index.md (hunk 1 above)
Comment (heyawhite) [reply site/en/docs/privacy-sandbox/overview/index.md:221]: That's just something I've observed previously -- if it's not actually breaking it, it's not a big deal. But the space before the `###` is what I was referring to.

```suggestion
### Get involved
```

**Review by heyawhite — COMMENTED** [review]
Some feedback to address before approval

File: site/en/docs/privacy-sandbox/fledge-api/index.md (hunk 2 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/fledge-api/index.md]: Done.

**Comment by nancymic2** [comment]
I can continue with the FAQs after we merge this ---i.e. the topics content etc...

**Pull request merged**